components/raftstore/src/store/peer.rs

```diff
@@ -4269,6 +4295,9 @@ where
             }
         }
 
+        self.pending_propose_instants
+            .push_back((propose_index, Instant::now()));
```

Comment:
use `coarse_now()`

Reply:
I think `Instant::now()` is fast enough. It occupies <0.05% CPU on the flamegraph in my testing. I worry `coarse_now()` may not have enough accuracy (the accuracy can be 10ms on some Linux distributions).